api-call.py

```diff
@@ -1,29 +1,226 @@
-import requests
+import datetime
 import json
 
+import pytz
+import requests
+import yaml
+
+
+def load_from_config():
+    """Loading important parameters from configuration file
+
+    :returns: The config object
+    """
+
+    with open('configuration.yaml', 'r') as file:
+        return yaml.safe_load(file)
+
+
+def log_error(error_message):
```

Comment:
In newer Python, we can define [typing](https://docs.python.org/3/library/typing.html) for functions and I believe it's a good practice that we should follow too.

Reply:
implemented.